test/allow_list.js

```diff
@@ -0,0 +1,280 @@
+const AllowListContract = artifacts.require("ValidatorSmartContractAllowList.sol");
+
+var fs = require('fs');
+var jsonFile = "build/contracts/ValidatorSmartContractAllowList.json";
+var parsed= JSON.parse(fs.readFileSync(jsonFile));
+var allowListContractAbi = parsed.abi;
+
+
+contract ("Account Ingress (no contracts registered)", (accounts) => {
+    const validators = accounts;
```

Comment:
Better to use a different set of validators to catch bugs where account and validator are mixed up.